### PR TITLE
fix(specs): multiple clients fixes

### PR DESCRIPTION
**Pull request by shortcuts**
## 🧭 What and Why

🎟 JIRA Ticket: https://algolia.atlassian.net/browse/DI-3071

### Changes included:

this pr fixes some of the issues identified in the client testing from @kombucha

### CODE REVIEW

**Review by Fluf22 — APPROVED** [review]
✅

File: specs/search/paths/keys/common/schemas.yml [hunk 1]
```diff
@@ -109,7 +109,7 @@ baseGetApiKeyResponse:
     createdAt:
       $ref: '../../../../common/responses/common.yml#/createdAtTimestamp'
   required:
-    - key
+    - value
```
[the remark this answers is not in the document]
Comment (shortcuts) [reply specs/search/paths/keys/common/schemas.yml:112]: I was shocked too idk how it went through?

[the remark this answers is not in the document]
Comment (millotp) [reply specs/search/paths/keys/common/schemas.yml:112]: actually there are some warnings like `Component: "attributeToUpdate" is never used.`

[the remark this answers is not in the document]
Comment (shortcuts) [reply specs/search/paths/keys/common/schemas.yml:112]: ah in the warnings we silence? I'll check that

**Review by Fluf22 — APPROVED** [review]
🆗

**Pull request merged**